data/plugins/calculated-property.ts

```diff
@@ -0,0 +1,16 @@
+import { Plugin } from "./index";
+
+export const CalculatedProperty: Plugin = {
+  name: "Calculated Property",
+  description:
+    "Grouparoo's Calculated Property plugin allows you to transform data from your warehouse before you export to various destinations such as CRMs, Customer Support Tools, and Marketing Tools.",
+  priority: 0,
+  slug: "calculated-property",
+  logo: "calculated-property.png",
+  primaryType: "source",
+  otherTypes: [],
+  category: "transform tool",
+  badge: "",
+  showLink: false,
+  packageName: "@grouparoo/calculated-property",
+};
```

Comment:
This file generates the manifest entry for calculated properties... it is not imported into the `/integrations` page at all but is still needed for the manifest.